src/components/TopicMenu/Topics/SectionPage.tsx

```diff
@@ -57,49 +59,181 @@ function SectionPage() {
+                    </CSSTransition>
                 ): data?.name === 'Conditions' ? (
+                                        <CSSTransition
+                    in={true}
+                    appear={true}
+                    timeout={1000}
+                    classNames="fade">
                     <ConditionsPage name={data?.name} splitDesc={splitDesc}  />
+                    </CSSTransition>
                 ): data?.name === 'Diseases' ? (
+                                        <CSSTransition
+                    in={true}
+                    appear={true}
+                    timeout={1000}
+                    classNames="fade">
                     <DiseasesPage name={data?.name} splitDesc={splitDesc}  />
+                    </CSSTransition>
                 ): data?.name === 'Madness' ? (
+                                        <CSSTransition
+                    in={true}
+                    appear={true}
+                    timeout={1000}
+                    classNames="fade">
                     <MadnessPage name={data?.name} splitDesc={splitDesc}  />
+                    </CSSTransition>
                 ): data?.name === 'Objects' ? (
+                                        <CSSTransition
+                    in={true}
+                    appear={true}
+                    timeout={1000}
+                    classNames="fade">
                     <ObjectsPage name={data?.name} splitDesc={splitDesc}  />
+                    </CSSTransition>
                 ): data?.name === 'Poisons' ? (
+                                        <CSSTransition
+                    in={true}
+                    appear={true}
+                    timeout={1000}
+                    classNames="fade">
                     <PoisonsPage name={data?.name} splitDesc={splitDesc}  />
+                    </CSSTransition>
                 ): data?.name === 'Traps' ? (
+                                        <CSSTransition
+                    in={true}
+                    appear={true}
+                    timeout={1000}
+                    classNames="fade">
                     <TrapsPage name={data?.name} splitDesc={splitDesc}  />
+                    </CSSTransition>
                 ): data?.name === 'Spellcasting' ? (
+                                        <CSSTransition
+                    in={true}
+                    appear={true}
+                    timeout={1000}
+                    classNames="fade">
                     <SpellCastingPage name={data?.name} splitDesc={splitDesc}  />
+                    </CSSTransition>
                 ): data?.name === 'Pantheons' ? (
+                                        <CSSTransition
+                    in={true}
+                    appear={true}
+                    timeout={1000}
+                    classNames="fade">
                     <PantheonsPage name={data?.name} splitDesc={splitDesc}  />
+                    </CSSTransition>
```

Comment:
I wonder if there is a way to minimize how many times we instantiate the CSSTransition component. What do you think about looking into a way to programmatically deciding which component we need to open within the CSS Transition instead of outside it. This would limit the number of times we type <CSSTransition
                    in={true}
                    appear={true}
                    timeout={1000}
                    classNames="fade">